src/backend/distributed/operations/shard_rebalancer.c

```diff
@@ -1274,6 +1293,8 @@ get_rebalance_progress(PG_FUNCTION_ARGS)
 				cstring_to_text(PlacementUpdateTypeNames[step->updateType]));
 			values[12] = LSNGetDatum(sourceLSN);
 			values[13] = LSNGetDatum(targetLSN);
+			values[14] = PointerGetDatum(
+				cstring_to_text(PlacementUpdateStatusNames[step->updateStatus]));
```

Comment:
`step->updateStatus` should be read and written using atomic operations. See changes in `step->progress` for an example.